pkg/web/templates/partials/user/user_list.html

```diff
@@ -1,4 +1,5 @@
-{{ $role := .AuthUserRole }}
+{{- $isAdmin := .IsAdmin -}}
```

Comment:
This has to be moved to the top since the `with` overwrites the local context with the user list.